openassessment/assessment/api/ai.py

```diff
@@ -67,7 +67,8 @@ def get_score(submission_uuid, requirements):
         requirements (dict): Not used.
 
     Returns:
-        A dictionary with the points earned and points possible.
+        A dictionary with the points earned, points possible, and
+        contributing_assessments information.
```

Comment:
I think this should mention ``staff_id`` even though it is always ``None`` in this case. It is returned as part of the dictionary.